### PR TITLE
New pr

### PR DESCRIPTION
**Pull request by brunoantognolli**
test to new PR

### CODE REVIEW

File: src/test/java/org/springframework/samples/petclinic/TestNoord.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.springframework.samples.petclinic;
+
+import org.junit.jupiter.api.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestNoord {
+	@Test
+	public void simpleTestTrue() {
```
Comment (andrzejsydor) [inline src/test/java/org/springframework/samples/petclinic/TestNoord.java:10]:
```suggestion
	void simpleTestTrue() {
```

File: src/test/java/org/springframework/samples/petclinic/TestNoord.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package org.springframework.samples.petclinic;
+
+import org.junit.jupiter.api.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestNoord {
```
Comment (andrzejsydor) [inline src/test/java/org/springframework/samples/petclinic/TestNoord.java:8]:
```suggestion
class TestNoord {
```

**Review by andrzejsydor — CHANGES_REQUESTED** [review]
Why so many commits?
What are the commit names about? -> could be improved

**Comment by mdeinum** [comment]
Please remove this PR, don't use a public repository for your own learning use your own fork instead!

**Pull request closed without merging**